package.json

```diff
@@ -59,6 +59,7 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.2",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Why is this being added if we already had it before (see `particle.js` prepending it to entry points).

Reply:
I think we replaced it with @babel/preset-env

Reply:
This should be removed since it is for version 6 of Babel and we are on version 7. @babel/env on line 60 at this PR is the package we need. It is being prepended in [particle.js](https://github.com/phase2/particle/blob/master/particle.js#L80).

Reply:
done :-)

Reply:
Why is this still in?